app/views/spectator_sport/dashboard/session_windows/details.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= link_to "Back to dashboard", "/spectator_sport_dashboard" %>
```

Comment:
Ugh, this again. Will fix soon based on your changes.

Reply:
I just merged my PR